README.md

```diff
@@ -181,84 +218,79 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
-didn't have to explicitly import it into this component. With one explicit import of
-that icon in `App.js`, and adding it to the library, we've managed to use
-it by name in multiple components.
+  didn't have to explicitly import it into this component. With one explicit
+  import of that icon in `App.js`, and adding it to the library, we've managed
+  to use it by name in multiple components.
 * We used the `"apple"`, `"microsoft"`, and `"google"` brand icons, which were
-never explicitly _individually_ imported, but they're available to us by
-name as strings because `fab` was added to our library in `App.js`, and
-`fab` includes all of those icons.
+  never explicitly _individually_ imported, but they're available to us by name
+  as strings because `fab` was added to our library in `App.js`, and `fab`
+  includes all of those icons.
 * We added the `fab` prefix to reference those brand icons.
 
 Adding a prefix—and the syntax we used to do it—are new. So what's
 going on here?
 
-First, recall when we introduced
-<span style="whitespace:nowrap;">`<FontAwesomeIcon icon="coffee"/>`</span>
-and learned that a prefix of `fas` was being added  to `"coffee"` by default.
+First, recall when we introduced `<FontAwesomeIcon icon="coffee"/>` and learned
+that a prefix of `fas` was being added to `"coffee"` by default.
 
-The `"check-square"` icon is getting a default prefix of `fas` here too,
-which is what we want, because that icon also lives in the
-<span style="whitespace:nowrap;">`@fortawesome/fontawesome-free-solid`</span>
-package.
+The `"check-square"` icon is getting a default prefix of `fas` here too, which
+is what we want, because that icon also lives in the
+`@fortawesome/free-solid-svg-icons` package.
 
 However, the `"apple"`, `"microsoft"`, and `"google"` brand icons live in the
-package <span style="whitespace:nowrap;">`@fortawesome/fontawesome-free-brands`</span>.
-So we need to specify a different prefix for them—not the default `fas`,
-but `fab`, for Font Awesome _Brand_.
+package `@fortawesome/free-brands-svg-icons`. So we need to specify a
+different prefix for them—not the default `fas`, but `fab`, for Font Awesome
+_Brand_.
 
 When specifying a prefix with an icon name, both are given as strings.
 
 Now, what about that syntax?
 
 The `icon` prop expects a single object:
+
 * It could be an icon object, like `{faCoffee}`.
 * It could a string object, like `"coffee"`.
+  (The curly braces around a string object supplied to a prop are optional,
+  so we've omitted them.)
+* Or it could be an `Array` of strings, where the first element is a prefix,
+  and the second element is the icon name: `{["fab", "apple"]}`
 
-    (The curly braces around a string object supplied to a prop are
-    optional, so we've omitted them.)
-* Or it could be an `Array` of strings, where the first element is a
-prefix, and the second element is the icon name: `{["fab", "apple"]}`
+### Processing `<i>` tags into `<svg>` using Font Awesome
```

Comment:
I think it might be helpful to state more specifically in this section that processing `<i>` tags is not the React Way. It's really the sans-React way. And the reason for mentioning that here is to say that, if you know that you want to be able to do both (a mixed mode), here's how you can enable that.

When I try to imagine myself in the shoes of a newbie reading this, it might make me think: "OK, this section is tell me that if I want _basic_ Font Awesome support, I need to enable it with `watch()`."

What I'd want her to understand is more like this:
-  if what you really want to do is use `<i>` and you're not interested in React, then you don't need this React component, just go use the basic SVG with JS method and `<i>` tags.
- if what you really want to do is use _React_, then you never need to use `<i>` tags. And because we assume that you're using the React component because you want to use Font Awesome in a way that optimized for React, we've disabled the `<i>` tag replacement stuff by default as an optimization.
- but if you know that you need to have both things going on at the same time, then this is how you'd enable the `<i>` tag autoReplace, etc.

Reply:
Oh nice point. Ok, let me take another shot at that.